plugins/modules/azure_rm_apimanagement_info.py

```diff
@@ -0,0 +1,273 @@
+        resource_group: myResourceGroup
+        service_name: myService
+    - name: Get the information of api
+      azure_rm_apimanagement_info:
+        resource_group: myResourceGroup
+        service_name: myService
+        api_id: testApi
+'''
+
+RETURN = '''
+api:
+  description:
+    - A list of dict results where the key is the name of the Api and the values are the facts for that Api.
+  returned: always
+  type: complex
+  contains:
+    api_name:
+      description: The api name provided by the user.
+      type: complex
+      contains:
+        id:
+          description:
+            - Resource ID.
+          returned: always
+          type: str
+          sample: null
+        name:
+          description:
+            - Resource name.
+          returned: always
+          type: str
+          sample: null
+        type:
+          description:
+            - Resource type for API Management resource.
+          returned: always
+          type: str
+          sample: null
+        properties:
+          description:
+            - Api entity contract properties.
+          returned: always
+          type: dict
+          sample: null
+
+'''
+
+from ansible_collections.azure.azcollection.plugins.module_utils.azure_rm_common_rest import GenericRestClient
+from ansible_collections.azure.azcollection.plugins.module_utils.azure_rm_common_ext import AzureRMModuleBaseExt
+from copy import deepcopy
+import time
+import json
+import re
+try:
+    from msrestazure.azure_exceptions import CloudError
+except ImportError:
+    # This is handled in azure_rm_common
+    pass
+
+class AzureApiManagementInfo(AzureRMModuleBaseExt):
+    def __init__(self):
+        self.module_arg_spec = dict(
+            resource_group=dict(
+                type='str',
+                required=True
+            ),
+            service_name=dict(
+                type='str',
+                required=True
+            ),
+            expand_api_version_set=dict(
+                type='bool'
+            ),
+            include_not_tagged_apis=dict(
+                type='bool'
+            ),
+            api_id=dict(
+                type='str'
+            )
+        )
+
+        self.resource_group = None
+        self.service_name = None
+        self.tags = None
+        self.expand_api_version_set = None
+        self.include_not_tagged_apis = None
+        self.api_id = None
+
+        self.results = dict(changed=False)
+        self.mgmt_client = None
+        self.state = None
+        self.url = None
+        self.status_code = [200]
+
+        self.query_parameters = {}
+        self.query_parameters['api-version'] = '2020-06-01-preview'
+        self.header_parameters = {}
+        self.header_parameters['Content-Type'] = 'application/json; charset=utf-8'
+
+        super(AzureApiManagementInfo, self).__init__(derived_arg_spec=self.module_arg_spec,
+                                                    supports_check_mode=True,
```

Comment:
159,160, 186, 192, 198,208, 209,210,211,212,213, 214,229, 230,231, 232, 233,234,235,250,251,252, 253,254,255 and 256  continuation line under-indented for visual indent